test/pytest.ini

```diff
@@ -18,8 +18,11 @@ log_file_level=debug
 log_file_format=%(asctime)s - %(module)-12s %(funcName)-15s [%(levelname)-8s] %(message)s
 log_file_date_format=%d.%m.%Y %H:%M:%S
 
-#pep8 plugin
-pep8ignore = E402 E501
+#flake8 plugin
+flake8-ignore = E402 E501 E722 W504 W605
 # E402 # import not at top
 # E501 # line too long
-# pep8maxlinelength = 99
\ No newline at end of file
+# E722 # do not use bare 'except'
+# W504 # line break after binary operator
+# W605 # invalid escape sequence
+# flake8-max-line-length = 99
```

Comment:
Hier sollten wir mal klären, welche Ausnahmen wir davon wirklich ignorieren wollen.
Oder ob es Sinn macht, diese zuzulassen und mittelfristig zu beheben.

Reply:
> 
> 
> Außerdem sollten wir den autom. Test noch um Py 3.9 anpassen, oder was meinst du?

Jo macht absolut sinn, hab ich mit rein gepackt, schauen ob alles ohne Probleme durchlaeuft

Reply:
Macht vermutlich sinn, die mittelfristig zu beheben, hatte die bis jetzt noch ignoriert da hier noch einige fehler anfallen, hauptsaechlich wegen den Datei-Headern.

W504 kommt in konflikt mit W503, da ist wohl irgendwas bei flake8 vehauen, oder es gibt keine "richtige" loesung.

E722 Sollten wir glaube ich moeglichst beheben, wobei der "catch all Exceptions" ja auch schon wichtig sein kann.

Reply:
"catch all" sollte schon nach Möglichkeit vermieden werden.
An den wenigen Stellen, wo es wirklich erwünscht ist kann man das auch mit einem FLAG direkt im Sourcecode abschalten: https://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html#in-line-ignoring-errors